content/spin/v2/key-value-store-tutorial.md

```diff
@@ -87,10 +97,20 @@ $ spin new http-go spin-key-value
 
 ## Configuration
 
-Good news - Spin will take care of setting up your key value store. However, in order to make sure your Spin application has permission to access the key value store, you must add the `key_value_stores = ["default"]` line in the `[[component]]` area of the `spin.toml` file. This line is necessary to communicate to Spin that a given component has access to the default key value store. A newly scaffolded Spin application will not have this line; you will need to add it. 
+Good news - Spin will take care of setting up your key value store. However, in order to make sure your Spin application has permission to access the key value store, you must add the `key_value_stores = ["default"]` line in the `[component.<app-name>]` section of the `spin.toml` file, for each component which needs access to the key-value store. This line is necessary to communicate to Spin that a given component has access to the default key value store. A newly scaffolded Spin application will not have this line; you will need to add it. 
+
+> Note: `[component.spin_key_value]` contains the name of the component. If you used a different name, when creating the application, this sections name would be different.
+
+```toml
+[component.spin_key_value]
+...
+key_value_stores = ["default"]
+...
+```
 
 >> Tip: You can choose between various store implementations by modifying [the runtime configuration](dynamic-configuration.md#key-value-store-runtime-configuration). The default implementation uses [SQLite](https://www.sqlite.org/index.html) within the Spin framework.
 
+<!-- TODO: Check this section! -->
```

Comment:
Yeah this has not changed

Reply:
Good - I see I misread this as "If one of your application's component cretes a new key/value STORE, ..."